.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2.1
+
+jobs:
+  verify:
+    docker:
+      - image: circleci:python/3.9
```

Comment:
We're not using Python here yet, but given some of the more complex logic described in #21, I'm guessing we may want to start doing so soon. If not, `cimg/base:stable` could be a reasonable alternative (currently maps to Ubuntu 20.04 and will track LTS).

Reply:
Let's use `cimg/* images here, since they are circleci's next gen images [1,2]. We have been using `cimg:python` in some areas of core (https://github.com/freedomofpress/securedrop/pull/5720) and they have proven to be reliable so far.

[1]: https://circleci.com/developer/images
[2]: https://circleci.com/docs/2.0/circleci-images/#next-generation-convenience-images

Reply:
Sounds good, done.